src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/CryptoConfig.cs

```diff
@@ -0,0 +1,443 @@
+                // ht.Add("http://www.w3.org/2001/04/xmldsig-more#hmac-ripemd160", HMACRIPEMD160Type);
+            }
+        }
+
+        public static void AddAlgorithm(Type algorithm, params string[] names)
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public static object CreateFromName(string name, params object[] args)
+        {
+            if (name == null)
+                throw new ArgumentNullException(nameof(name));
+
+            Type retvalType = null;
+
+            // We allow the default table to Types and Strings
+            // Types get used for types in .Algorithms assembly.
+            // strings get used for delay-loaded stuff in other assemblies such as .Csp.
+            object retvalObj;
+            if (DefaultNameHT.TryGetValue(name, out retvalObj))
+            {
+                if (retvalObj is Type)
+                {
+                    retvalType = (Type)retvalObj;
+                }
+                else if (retvalObj is string)
+                {
+                    retvalType = Type.GetType((string)retvalObj, false, false);
+                    if (retvalType != null && !retvalType.IsVisible)
+                    {
+                        retvalType = null;
+                    }
+                }
+                else
+                {
+                    Debug.Fail("Unsupported Dictionary value:" + retvalObj.ToString());
+                }
+            }
+
+            // Maybe they gave us a classname.
+            if (retvalType == null)
+            {
+                retvalType = Type.GetType(name, false, false);
+                if (retvalType != null && !retvalType.IsVisible)
+                {
+                    retvalType = null;
+                }
+            }
+
+            // Still null? Then we didn't find it.
+            if (retvalType == null)
+            {
+                return null;
+            }
+
+            // Locate all constructors.
+            MethodBase[] cons = retvalType.GetConstructors(ConstructorDefault);
+            if (cons == null)
+            {
+                return null;
+            }
+
+            if (args == null)
+            {
+                args = new object[] { };
+            }
+
+            List<MethodBase> candidates = new List<MethodBase>();
+            for (int i = 0; i < cons.Length; i++)
+            {
+                MethodBase con = cons[i];
+                if (con.GetParameters().Length == args.Length)
+                {
+                    candidates.Add(con);
+                }
+            }
+
+            if (candidates.Count == 0)
+            {
+                return null;
+            }
+
+            cons = candidates.ToArray();
+
+            // Bind to matching ctor.
+            object state;
+            ConstructorInfo rci = Type.DefaultBinder.BindToMethod(
```

Comment:
I verified Binder.BindToMethod and ConstructorInfo.Invoke are in ns2.0: https://raw.githubusercontent.com/dotnet/standard/master/docs/versions/netstandard2.0_ref.md